ui/projekt-monsterz-front/src/app/header/header.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, EventEmitter, Output} from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.css']
+})
+export class HeaderComponent {
+  @Output() newContentChangeEvent = new EventEmitter<string>();
```

Comment:
Stylistically, I'm used to decorators being given their own line, so this is interesting to see. I don't mind it here, especially because it's empty.